webpack/assets/javascripts/react_app/components/common/Date/RelativeDate.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedRelative } from 'react-intl';
+
+class RelativeDate extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let date, title;
+
+    if (this.props.date) {
+      date = this.props.date;
+
+      title = this.props.intl.formatDate(date, {
+        day: '2-digit',
+        month: 'short',
+        hour: '2-digit',
+        minute: '2-digit',
+        hour12: false
```

Comment:
This comment also applies to the `AbsoluteDate` class above but I would think that all of these should be set based on the user's locale and not hardcoded.